website/src/roadmap/spark-runner.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: section
+title: "Spark Runner Roadmap"
+permalink: /roadmap/spark-runner/
+section_menu: section-menu/roadmap.html
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Apache Spark Runner Roadmap
```

Comment:
@jbonofre @echauchot @iemejia anything to share here? I did not copy over the Spark 2 item because I think maybe it is resolved?

Reply:
@kennknowles 
I think it is too early to share but, we have the targets bellow:
- short to middle term: fill the capability matrix gaps (among others state/timer in streaming mode, PCollectionView in streaming mode, etc...). I just started to take a look 
- middle to long term: migrate from dstream to  dataset/dataframes to support structured streaming and leverage schema pcollections. But this is just ideas, we have not yet started to think about it.